### PR TITLE
ui: add a table with logs to the (currently empty) logs page

### PR DESCRIPTION
**Pull request by maxlang**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/8572)

### CODE REVIEW

**Comment by maxlang** [comment]
screen shot 2016-08-15 at 10 15 27 pm

**Comment by mrtracy** [comment]
Review status: 0 of 7 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[ui/app/containers/nodeLogs.tsx, line 13 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/8572#-KPOV3VWIcmUFTy9ZD0p:-KPOV3VWIcmUFTy9ZD0q:b4orob2) ([raw file](https://github.com/cockroachdb/cockroach/blob/a1ebba027012df2dc01130006ef20cf2d186a88a/ui/app/containers/nodeLogs.tsx#L13)):*
> ```XML
> import { FormatSeverity } from "../util/format";
> 
> import { SortableTable /*, SortableColumn, SortSetting*/} from "../components/sortabletable";
> ```

Leftover commented-out code

---

*[ui/app/containers/nodeLogs.tsx, line 52 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/8572#-KPOVJY6pl2XFoRdm_y-:-KPOVJY6pl2XFoRdm_y0:b-d7bedk) ([raw file](https://github.com/cockroachdb/cockroach/blob/a1ebba027012df2dc01130006ef20cf2d186a88a/ui/app/containers/nodeLogs.tsx#L52)):*
> ```XML
>       return <div className="logs-table">
>         <SortableTable count={logEntries.length}>
>           { columns }
> ```

This will result in a merge skew error for one of us, i've changed this interface in an outstanding PR.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/8572)*

**Comment by maxlang** [comment]
Review status: 0 of 7 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[ui/app/containers/nodeLogs.tsx, line 13 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/8572#-KPOV3VWIcmUFTy9ZD0p:-KPOkf64Z6sBNEDR01tO:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/a1ebba027012df2dc01130006ef20cf2d186a88a/ui/app/containers/nodeLogs.tsx#L13)):*
<details><summary><i>Previously, mrtracy wrote…</i></summary>
> Leftover commented-out code
</details>
Done.

---

*[ui/app/containers/nodeLogs.tsx, line 52 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/8572#-KPOVJY6pl2XFoRdm_y-:-KPOkg9QJuT9D2xAz5JN:bnghwkb) ([raw file](https://github.com/cockroachdb/cockroach/blob/a1ebba027012df2dc01130006ef20cf2d186a88a/ui/app/containers/nodeLogs.tsx#L52)):*
<details><summary><i>Previously, mrtracy wrote…</i></summary>
> This will result in a merge skew error for one of us, i've changed this interface in an outstanding PR.
</details>
Up to you who merges first.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/8572)*

**Comment by mrtracy** [comment]
LGTM, but I would like to merge first.

File: ui/app/util/api.ts [hunk 1]
```diff
@@ -167,3 +176,15 @@ export function getCluster(): Promise<ClusterResponseMessage> {
 export function getTableStats(req: TableStatsRequest): Promise<TableStatsResponseMessage> {
   return Fetch(serverpb.TableStatsResponse, `${API_PREFIX}/databases/${req.database}/tables/${req.table}/stats`);
 }
+
+// TODO (maxlang): add filtering
+// TODO: switch to using Fetch once #8569 is fixed
```
Comment (tamird) [inline ui/app/util/api.ts]: #8569 was fixed by https://github.com/cockroachdb/cockroach/pull/8576 yesterday.

**Pull request merged**